source/2020-03-20-the-ember-times-issue-140.md

```diff
@@ -9,19 +9,20 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
-
+Read about testing best practices on the Ember Testing Guide 📗,
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Ember Testing Guide v1 📗](https://twitter.com/PoslinskiNet/status/1239503392386568192)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Dawid Pośliński (@PoslinskiNet)](http://github.com/PoslinskiNet/) wrote about testing best practices and which addons to use in the newly released [Ember Testing Guide v1](https://selleo.com/ebook/ember). The testing guide goes through why we need tests to begin with. What benefits does testing give us and what are the gaps that are still present.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Following that, the testing guide discusses testing best practices in how we architect our testing code. It also goes through how to use the right kind of test such as if it should be a unit, integration, acceptance, and even manual or visual regression testing. These principles are the building blocks to constructing effective tests which the guide goes through.
```

Comment:
Because I haven't read the guide yet, I wasn't sure if, by "testing best practices," whether you meant (1) the guide _tests_ best practices or (2) best practices for testing. I assumed the latter and thought we could omit the word "testing" (since the word appears many times in this paragraph).
```suggestion
Following that, the testing guide discusses best practices in how we architect our testing code. It also goes through how to use the right kind of test. Should the test be unit, integration, acceptance, visual, or even manual? These principles are the building blocks to constructing effective tests.
```